docs/releasing.md

```diff
@@ -1,8 +1,36 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
+
+### Release manager
+
+Rather than a rotation, release manager is appointed. However if the appointed manager is absent, the next engineer in the following list owns it:
```

Comment:
Who appoints the release manager?

Reply:
I'm proposing it's basically brendan, then me, then patrick, based on availability.

Reply:
Got it. Not sure if I grokked that from the text, suggested revision: "release manager is appointed, according to the list below. However, if the appointed manager is absent, the next engineer in line in the list would own it."